library/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources>
+    <string name="about_page_description" translatable="false" >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vehicula ligula tortor, in pellentesque dui congue ac. Vestibulum lacinia urna magna, quis cursus ante iaculis nec. Nunc ut ligula sit amet odio ultrices euismod vel vel purus. Sed lacus mauris, aliquam et placerat id</string>
+
+    <string name="about_contact_us">Kontakt</string>
+    <string name="about_instagram">Śledź nas na Instagramie</string>
+    <string name="about_facebook">Polub nas na Facebooku</string>
+    <string name="about_play_store">Oceń nas w Sklepie Play</string>
+    <string name="about_youtube">Zobacz nas na Youtube</string>
```

Comment:
With "kanał" will be ok too

20 sie 2016 9:33 PM "Zenedith" <notifications@github.com> napisał(a):

> In library/src/main/res/values-pl/strings.xml
> <https://github.com/medyo/android-about-page/pull/65#discussion_r75583636>
> :
>
> > @@ -0,0 +1,13 @@
> > +<resources>
> > +    <string name="about_page_description" translatable="false" >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vehicula ligula tortor, in pellentesque dui congue ac. Vestibulum lacinia urna magna, quis cursus ante iaculis nec. Nunc ut ligula sit amet odio ultrices euismod vel vel purus. Sed lacus mauris, aliquam et placerat id</string>
> > +
> > +    <string name="about_contact_us">Kontakt</string>
> > +    <string name="about_instagram">Śledź nas na Instagramie</string>
> > +    <string name="about_facebook">Polub nas na Facebooku</string>
> > +    <string name="about_play_store">Oceń nas w Sklepie Play</string>
> > +    <string name="about_youtube">Zobacz nas na Youtube</string>
>
> Zobacz nas na Youtube -> Zobacz nasz kanał na Youtube?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/medyo/android-about-page/pull/65/files/682a203efdd6f492237dffb437f5a39611679118#r75583636>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARTxGd4EoT50Eu2a3M9e0O2tKEbW-cGqks5qh1aYgaJpZM4JpGo8>
> .
>